Assets/Scripts/Characters/Projectile.cs

```diff
@@ -21,6 +24,12 @@ public class Projectile : MonoBehaviour {
 	// Update is called once per frame
 	void Update () {
 
+
+
+        gameObject.transform.position += orignalDirection * speed * Time.timeScale;
```

Comment:
It's not supposed to be a dumb projectile, it's supposed to track *slightly* rather than perfectly.